Automattic/wp-calypso
# Add experience field

A continuation of #19693. This is the final PR that add's the last question `How comfortable are you with creating a website?`. This question is only visible to new users that haven't created an account yet. The results from this question are stored in a tracks event. 

![experience](https://user-images.githubusercontent.com/6981253/33221733-d3218d4e-d11f-11e7-9d8f-003471ae684f.gif)

## Test not logged in
- Visit `/start/segment` in an incognito window
- Pick one of the options under `How comfortable are you with creating a website?
- Click on continue to proceed to the domain step.
- Click `Back` to return to the new segmentation step and see your selection still highlighted.

## Test logged in
- Visit `/start/segment`logged in
- You shouldn't see the `How comfortable are you with creating a website?` question 

@markryall and @taggon can you take a look?
